index.html

```diff
@@ -2282,6 +2283,10 @@
           <a>receiving user agent</a>.
         </p>
         <p>
+          A <a>PresentationReceiver</a> object is associated with a
+          <a>PresentationConnectionList</a> object when it is created.
```

Comment:
The second "it" is a little ambiguous.  Suggested rephrasing:

When the PresentationReceiver object is created, a PresentationConnectionList object is also created and associated with that PresentationReceiver object.